CMakeLists.txt

```diff
@@ -355,7 +355,7 @@ IF (CMAKE_COMPILER_IS_GNUCXX)
     #
     # This warning is more effective with link time optimization, where the
     # information about the class hierarchy graph is more complete.
-    TARGET_COMPILE_OPTIONS(${CMAKE_PROJECT_NAME} PRIVATE -Wsuggest-final-types)
+    TARGET_COMPILE_OPTIONS(${CMAKE_PROJECT_NAME} PRIVATE )#-Wsuggest-final-types)
```

Comment:
Locale compilation specific, must not be send to the master branch. Moreover must comment the entire line (in local)